school_evaluation/models/school_evaluation.py

```diff
@@ -163,3 +164,20 @@ class RatingRating(models.Model):
     point = fields.Integer('Rating in points', required=True,
                            help="Points")
     rating = fields.Char('Remarks', required=True)
+
+
+class StudentExtend(models.Model):
+    _inherit = 'student.student'
+
+    @api.multi
+    def set_alumni(self):
+        '''Override method to set active false student evaluation when
+        student is set to alumni'''
+        for rec in self:
+            student_eval = self.env['school.evaluation'].\
+                search([('type', '=', 'student'),
+                        ('student_id', '=', rec.id)])
+            if student_eval:
+                for data in student_eval:
```

Comment:
@Arpit-SerpentCS Remove the for loop. You can achieve same result without for loop.